scripts/lib/map-helper-functions.js

```diff
@@ -2,6 +2,9 @@
   var mapHelperFunctions = {};
 
   mapHelperFunctions.zeroPad = function zeroPad(districtID) {
+    if (districtID.length === 3) {
+      return districtID.substring(1,districtID.length);
+    }
```

Comment:
are there districts that are zeropadded to 3 digits? Is that what this is for?

Reply:
I believe most, if not all, of the districts are zero padded. If I remember the dataset I used to populate db with they were just strings that I didn't convert. I didn't convert because I think some states already have 80 seats filled so given a couple more redistricting periods, it would be possible to push 100.

Reply:
yes, but thanks for checking this - I just realized I need to update it to check for a non-zero at the first digit (I made this modification for state legislatures, most of which look like this in the data: (003, 006, 014, etc.)

Reply:
I may be misunderstanding here, but there are quite a few states with triple-digit numbers of legislators. (Not these first four--part of the reason we chose them. But certainly down the line).